browser/src/robocorp/browser/cli.py

```diff
@@ -1,10 +1,25 @@
 # ruff: noqa: E501
 import argparse
+import logging
 import sys
 
 from ._engines import install_browser
 from ._types import BrowserEngine, InstallError
 
+log = logging.getLogger(__name__)
+
+# Use certificates from native storage (if `truststore` installed)
+if sys.version_info >= (3, 10):
+    try:
+        import truststore  # type: ignore
+
+        truststore.inject_into_ssl()
+        log.info(
+            "Truststore injection done, using system certificate store to validate HTTPS."
+        )
```

Comment:
While the logging does the job, do you see as an improvement if we'd try to use our own logger instead and import `robocorp.log` for that? Then use `log.info` & `log.warn` for the same, thing which might render better in the output _log.html_ file.

Or even better, adding a `log.console_message(...)` as well so we'd see it in both the console output and the log file, although this might add some duplicate messages in the output log file. Test it out and take a decision by yourself here on what you'd think it will render best!

The **robocorp-tasks** package already depends on **robocorp-log**.